ogame/__init__.py

```diff
@@ -550,7 +550,14 @@ def submit_marketplace(self, offer, price, range, id):
         quantity = None
         priceType = None
         price_form = None
-        self.session.get(self.index_php + 'page=ingame&component=marketplace&tab=overview&cp={}'.format(id))
+        resp = self.session.get(
```

Comment:
resp is normaly gets written out as response. just to be consistent with the already excisting code
see line 586